modules/WebcastPolls/resources/webcastPolls.js

```diff
@@ -394,6 +556,29 @@
                 _this.log("done syncing current poll state");
             }
         },
+		getUserID: function () {
```

Comment:
this used to be in another file (UserProfile) not sure why you duplicated it... it used to exits in userProfile memeber